.github/workflows/test.yml

```diff
@@ -155,10 +155,11 @@ jobs:
           path: |
             libs/ui/tests/react.spec.js-snapshots/
             libs/ui/test-results/
-      - name: Test coverage
-        run: npx codecov -F test
-        env:
-          CODECOV_TOKEN: ${{secrets.CODECOV_TOKEN}}
+      - uses: codecov/codecov-action@v3
+        if: always()
```

Comment:
Is `if: always()` necessary ? (sorry I am a newbie)

I don't see it in their doc here: https://github.com/codecov/codecov-action

Reply:
it will un the action even if the previous steps fail

Reply:
Ok, thanks !